.env.test

```diff
@@ -8,6 +8,7 @@ NODE_ENV=test
 PORT=5000
 POSITRON_URL=http://localhost:5002
 PREDICTION_URL=https://live.artsy.net
+RECAPTCHA_KEY=6LdvSaIUAAAAAD92qOsXbAEaBpBfLmWfd9_T-J59
```

Comment:
This is a public key created specifically for testing

Reply:
If tests would pass without this, I'd leave it out. (Or if a key is needed for certain tests, I'd stub it if possible.) I thought of the "test" key more as something we'd want to use in non-production envs (like staging and development), not necessarily in CI. Just let me know if I'm misunderstanding this.